docs/user_guide.rst

```diff
@@ -6,40 +6,28 @@
 User guide
 ==========
 
-If you are new to rlberry, see :ref:`the quickstart<quick_start>` documentation.
+If you are new to rlberry, check the :ref:`Tutorial` below and the :ref:`the quickstart<quick_start>` documentation.
 In the quick start, you will learn how to set up an experiment and evaluate the
 efficiency of different agents.
 
-For more informations see the other tutorials presented in this user guide and
-:ref:`the gallery of examples<examples>`.
+For more information see :ref:`the gallery of examples<examples>`.
 
 
-Basic agent management
-======================
+Tutorial
+=========
 
-.. toctree::
-  :maxdepth: 1
+The tutorials below will present to you the main functionalities of ``rlberry`` in a few minutes.
 
-  basics/quick_start.rst
-  basics/create_agent.rst
-  basics/experiment_setup.rst
+- :ref:`Install rlberry <installation>`.
+- :ref:`Create an agent <create_agent>`.
+- :ref:`Evaluate an agent and optimize its hyperparameters <evaluate_agent>`.
+- :ref:`Compare different agents <compare_agents>`.
+- :ref:`Setup and run experiments using yaml config files <experiment_setup>`.
+- :ref:`Quick start: setup an experiment and evaluate different agents <quick_start>`.
+- :ref:`Seeding & Reproducibility <seeding>`.
 
-Presentation of agents
-======================
 
-todo
-
-Comparing agents
-================
-
-.. toctree::
-
-  basics/evaluate_agent.rst
-  basics/compare_agents.rst
-  basics/seeding.rst
-
-
-How to contribute ?
+Contributing
```

Comment:
Ok, for now I'll change it back to how it was!

Reply:
After some research : this is a inherent behavior of rst.This can't really be changed. However we can target contributing doc and not the title using the `doc` keyword : 
```


Contributing
============

If you want to contribute to rlberry, check out :doc:`the contribution guidelines<contributing>`.
```
This would work, but on the other hand, all the :ref: links to contributing will send to the User Guide, we would have to change ref to doc in all the links to contributing (meaning the link in the User guide and the link in index.rst)